taskcluster/scripts/pyenv-setup

```diff
@@ -13,10 +15,11 @@ eval "\$(pyenv init -)"
 eval "\$(pyenv virtualenv-init -)"
 EOF
 source /builds/worker/.bashrc
-git clone https://github.com/momo-lab/xxenv-latest.git "$(pyenv root)"/plugins/xxenv-latest
+
 for i in ${1//,/ }
 do
-    pyenv latest install $i
+    version=$(pyenv latest --known $i)
+    pyenv install $version
```

Comment:
Nice, I see pyenv logs out what version (including the patch number) gets installed[1]:

```
Downloading Python-3.10.8.tar.xz...
-> https://www.python.org/ftp/python/3.10.8/Python-3.10.8.tar.xz
Installing Python-3.10.8...
Installed Python-3.10.8 to /builds/worker/.pyenv/versions/3.10.8
Downloading Python-3.9.15.tar.xz...
-> https://www.python.org/ftp/python/3.9.15/Python-3.9.15.tar.xz
Installing Python-3.9.15...
Installed Python-3.9.15 to /builds/worker/.pyenv/versions/3.9.15
Downloading Python-3.8.15.tar.xz...
-> https://www.python.org/ftp/python/3.8.15/Python-3.8.15.tar.xz
Installing Python-3.8.15...
Installed Python-3.8.15 to /builds/worker/.pyenv/versions/3.8.15
Downloading Python-3.7.15.tar.xz...
-> https://www.python.org/ftp/python/3.7.15/Python-3.7.15.tar.xz
Installing Python-3.7.15...
```

[1] https://firefox-ci-tc.services.mozilla.com/tasks/ZtcrXQW9SSmadkjrm9sqkQ/runs/0/logs/public/logs/live.log#L683